### PR TITLE
X Axis Labels 

### PR DESCRIPTION
**Pull request by PaulHax**
image

- [x] Still have to change visilbility on hover state of control points? 
- [x] I kind of want to show the range when zooming too?

Also:
* Big perf speedup from debouncing background render
* Line connecting color range only show when hovering on center color range point.  This avoids the line overlaping on X axis labels.

### CODE REVIEW

File: lib/AxisLabels.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        color: black;
+        position: absolute;
+        background-color: white;
+        border-style: solid;
+        border-color: black;
+        border-width: 1px;
+        font-size: ${FONT_SIZE}px;
+        padding: 2px 6px;
+        box-sizing: border-box;
+        transition: opacity 0.2s ease-in-out;
```
Comment (thewtex) [inline lib/AxisLabels.ts]: 0.2 -> 0.1 for practical responsiveness

**Review by thewtex — CHANGES_REQUESTED** [review]
Love. It.

Minor change requested inline.

File: lib/AxisLabels.ts (hunk 1 above)
Comment (PaulHax) [reply lib/AxisLabels.ts]: Yes, better, fixed.

**Review by thewtex — APPROVED** [review]
:green_heart:

**Pull request merged**